[2024-05-29T19:42:02.011+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.extract_user manual__2024-05-29T19:41:11.590015+00:00 [queued]>
[2024-05-29T19:42:02.048+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.extract_user manual__2024-05-29T19:41:11.590015+00:00 [queued]>
[2024-05-29T19:42:02.049+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:42:02.050+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-05-29T19:42:02.051+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:42:02.099+0000] {taskinstance.py:1383} INFO - Executing <Task(SimpleHttpOperator): extract_user> on 2024-05-29 19:41:11.590015+00:00
[2024-05-29T19:42:02.126+0000] {standard_task_runner.py:55} INFO - Started process 6023 to run task
[2024-05-29T19:42:02.137+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '1_dag', 'extract_user', 'manual__2024-05-29T19:41:11.590015+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/1_dag.py', '--cfg-path', '/tmp/tmpcipvkx5r']
[2024-05-29T19:42:02.147+0000] {standard_task_runner.py:83} INFO - Job 27: Subtask extract_user
[2024-05-29T19:42:02.392+0000] {task_command.py:376} INFO - Running <TaskInstance: 1_dag.extract_user manual__2024-05-29T19:41:11.590015+00:00 [running]> on host 4b56679d13b8
[2024-05-29T19:42:02.926+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_dag
AIRFLOW_CTX_TASK_ID=extract_user
AIRFLOW_CTX_EXECUTION_DATE=2024-05-29T19:41:11.590015+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-29T19:41:11.590015+00:00
[2024-05-29T19:42:02.942+0000] {http.py:121} INFO - Calling HTTP method
[2024-05-29T19:42:03.076+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-05-29T19:42:03.081+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-05-29T19:42:03.453+0000] {http.py:125} INFO - {"results":[{"gender":"male","name":{"title":"Mr","first":"Oğuzhan","last":"Örge"},"location":{"street":{"number":9314,"name":"Necatibey Cd"},"city":"Edirne","state":"Tokat","country":"Turkey","postcode":29531,"coordinates":{"latitude":"71.5726","longitude":"-163.1430"},"timezone":{"offset":"+11:00","description":"Magadan, Solomon Islands, New Caledonia"}},"email":"oguzhan.orge@example.com","login":{"uuid":"cffe84b3-8b7d-4dfc-b774-5d549dff9193","username":"heavyfish420","password":"integra","salt":"GTDZZ7Vo","md5":"fd4f2381a8b5b2e8e19a76f7c60aa9b2","sha1":"d00c4fb7a3496c3b88f8cef18ec4a449d12af8af","sha256":"1f30215adeffb4e0ef68f39de3309db5e287a294bf8aa4abcdf04ec998539b9a"},"dob":{"date":"1993-01-29T03:44:56.446Z","age":31},"registered":{"date":"2008-08-24T16:57:34.185Z","age":15},"phone":"(689)-960-5257","cell":"(466)-396-7636","id":{"name":"","value":null},"picture":{"large":"https://randomuser.me/api/portraits/men/28.jpg","medium":"https://randomuser.me/api/portraits/med/men/28.jpg","thumbnail":"https://randomuser.me/api/portraits/thumb/men/28.jpg"},"nat":"TR"}],"info":{"seed":"d9a918b6337702f5","results":1,"page":1,"version":"1.4"}}
[2024-05-29T19:42:03.461+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/http/operators/http.py", line 132, in execute
    return self.response_filter(response, **kwargs)
  File "/opt/airflow/dags/1_dag.py", line 58, in <lambda>
    response_filter = lambda response: json.load(response.text),
  File "/usr/local/lib/python3.7/json/__init__.py", line 293, in load
    return loads(fp.read(),
AttributeError: 'str' object has no attribute 'read'
[2024-05-29T19:42:03.548+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=1_dag, task_id=extract_user, execution_date=20240529T194111, start_date=20240529T194202, end_date=20240529T194203
[2024-05-29T19:42:03.631+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 27 for task extract_user ('str' object has no attribute 'read'; 6023)
[2024-05-29T19:42:03.723+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-05-29T19:42:03.839+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
